### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by edgarlanting**
While reading the readme/this repo I noticed a small typo: `thourgh` which should be changed to `through`, no big deal of course but just thought I'd mention.

### CODE REVIEW

**Comment by edgarlanting** [comment]
@ArmandGrillet Just openend this PR to change a small typo in the README.md, no biggie but since I don't have access to this repo I just thought I'd ask you for a review.

**Comment by edgarlanting** [comment]
> Thanks for the PR, from the diff it looks like the last line got his newline removed which is not intended. Can you please change that? I'll then merge the change.
> 
> PS: If you use VS Code, the parameter to always have newlines is `"files.insertFinalNewline": true`.

Thanks @ArmandGrillet odd think indeed but I've added a newline at the end of the file again.

File: README.md [hunk 1]
```diff
@@ -108,4 +108,5 @@ Tags are released to:
 
 ## Contributing
 
-Contributions are always welcome! Please refer to our [contributing guidelines](CONTRIBUTING.md).
\ No newline at end of file
+Contributions are always welcome! Please refer to our [contributing guidelines](CONTRIBUTING.md).
+
```
Comment (bamarni) [inline README.md]: Now there is an extra newline :sweat_smile: 

I'll leave it to Armand as he noticed it but should still be fine to merge, the typo fix is more important here.

Comment (edgarlanting) [reply README.md]: I've added the additional setting to VsCode to fix the newline thing, Thanks @bamarni for the review.

**Pull request merged**